datasets/mx/deputies/crawler.py

```diff
@@ -0,0 +1,66 @@
+import re
+import requests
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+def crawl_item(input_dict: dict, context: Context):
+
+    entity = context.make("Person")
+    entity.id = context.make_id(input_dict["NombreCompleto"])
+
+    entity.add("lastName", input_dict.pop("Nombre"))
+    entity.add("firstName", input_dict.pop("PrimerApellido"))
+    entity.add("name", input_dict.pop("NombreCompleto"))
+
+    if input_dict["Telefono"] is not None:
+        entity.add("phone", "+52"+input_dict.pop("Telefono"))
+
+    entity.add("email", input_dict.pop("Correo"))
+
+    position = h.make_position(context, "Member of the Chamber of Deputies", country="mx")
+    categorisation = categorise(context, position, is_pep=True)
+
+    occupancy = h.make_occupancy(
+            context,
+            entity,
+            position,
+            True,
+            categorisation=categorisation,
+        )
+
+    if occupancy is None:
+        return
+
+    context.emit(entity, target=True)
+    context.emit(position)
+    context.emit(occupancy)
+    
+
+def crawl(context: Context):
+
+    HEADER = {
+    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; rv:109.0) Gecko/20100101 Firefox/115.0',
+    'Accept': 'application/json, text/plain, */*',
+    'Accept-Language': 'en-US,en;q=0.5',
+    'Referer': 'https://web.diputados.gob.mx/',
+    'Content-Type': 'application/json',
+    'Origin': 'https://web.diputados.gob.mx',
+    'Connection': 'keep-alive',
+    'Sec-Fetch-Dest': 'empty',
+    'Sec-Fetch-Mode': 'cors',
+    'Sec-Fetch-Site': 'same-site',
```

Comment:
can we drop any of these headers without breaking things? I suspect accept language and the sec- headers aren't needed. also connection.

Reply:
In fact we don't even need to send any headers. I copied the curl command from the website and it was already with this headers, but looks like they are not necessary

Reply:
nice. Try and strip out anything from the code that isn't necessary. That makes identifying issues and maintenance in future easier.